client/src/Movies/Movie.js

```diff
@@ -11,10 +11,11 @@ export default class Movie extends Component {
 
   componentDidMount() {
     // change this line to grab the id passed on the URL
-    const id = 1;
-    this.fetchMovie(id);
+    const { id } = this.props.match.params;
```

Comment:
Object destructuring, that's really nice. Did you know you can do something like this as well? Try to find out how it works. Check MDN.

```js
const { 
  props: {
    match:  {
      params: { id }
    }
  }
} = this;
```

Not always useful to fully destructure but might come in handy sometime.